sdks/python/apache_beam/runners/portability/flink_runner.py

```diff
@@ -43,7 +43,13 @@ def default_job_server(self, options):
 class FlinkRunnerOptions(pipeline_options.PipelineOptions):
   @classmethod
   def _add_argparse_args(cls, parser):
-    parser.add_argument('--flink_master', default='[local]')
+    parser.add_argument('--flink_master',
+                        default='[auto]',
```

Comment:
`[local]` is also referenced at https://github.com/apache/beam/blob/release-2.17.0/sdks/python/apache_beam/runners/portability/flink_runner.py#L36

Reply:
This looks like it is for testing? That looks ok.